packages/ui/src/components/more-actions-tooltip.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react'
+import { Link } from 'react-router-dom'
+
+import { Button } from '@components/button'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuTrigger
+} from '@components/dropdown-menu'
+import { Icon } from '@components/icon'
+import { cn } from '@utils/cn'
+
+export interface ActionData {
+  to?: string
+  title: string
+  onClick?: () => void
+  isDanger?: boolean
+}
+
+export interface MoreActionsTooltipProps {
+  actions: ActionData[]
+}
+
+/**
+ * A component for displaying a button that, when clicked, shows a tooltip with a list of actions.
+ */
+export const MoreActionsTooltip: FC<MoreActionsTooltipProps> = ({ actions }) => {
```

Comment:
this kind of a component brings up interesting thoughts.

while this is useful to standardise all context menus, it also limits customisation which we get from composing lower layer components directly.

for eg. this component doesn't allow for icons, keyboard shortcuts or checkbox/radio items to be shown. Whereas the underlying DropdownItem component supports that. This means over time as requirements change, we will end up with multiple versions of this Tooltip component, with different features.

Hard to say which approach is better. cc @knagurski thoughts?